boards/riscv/qemu_riscv32/Kconfig.defconfig

```diff
@@ -8,7 +8,7 @@ config BOARD
 	default "qemu_riscv32_xip" if BOARD_QEMU_RISCV32_XIP
 	default "qemu_riscv32_smp" if BOARD_QEMU_RISCV32_SMP
 
-config COMPRESSED_ISA
+config RISCV_ISA_EXT_C
 	default y
```

Comment:
I do not think defconfig is the right place to do this, let alone in the board defconfig:

1. These are really "hardware properties" encoded in the Kconfig scheme, and these "properties" are not supposed to be mutable.
2. If any, this should be selected by `SOC_RISCV_VIRT` (no defconfig, but a hard select), which by the way is a SoC level Kconfig and not a board level Kconfig.

Reply:
agree. Again, this predates this PR. I'll try to fix this.